packages/ozone-typescript-client/src/ozoneClient/ozoneClientImpl.ts

```diff
@@ -299,7 +300,7 @@ export class OzoneClientImpl extends StateMachineImpl<ClientState> implements Oz
 		return new Promise<void>((resolve, reject) => {
 			/* FIXME AB Something is wrong here. The promise resolve or reject method should always be called but it is not the case */
 			const query = '?ozoneSessionId=' + this.authInfo!.sessionId
-			const ws = new WebSocket(this._config.webSocketsURL + query)
+			const ws = new SockJS(this._config.webSocketsURL + query)
```

Comment:
We could use template string instead of string concat
```suggestion
			const ws = new SockJS(`${this._config.webSocketsURL}?ozoneSessionId=${this.authInfo!.sessionId}`)
```